src/content/facts/09-cannibalism.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Sex sometimes turns into cannibalism.
+source: BBC Earth
+sourceUrl: http://www.bbc.com/earth/story/20150223-mysteries-of-cannibal-octopus-sex
+image: /images/valvos.jpg
+imageSource: https://www.oldbookillustrations.com/illustrations/valvos/
+---
+
+Octopus sex, as you might anticipate from a creature with so many arms, is a bit of an odd proposition. For males, it's also quite a dangerous one.
+
+Since female octopuses are often larger—and most octopuses are decidedly anti-social—there's always a chance that his lady partner will get bored and/or hungry mid-coitus, switching gears from "romance" to "having a delightful octopus snack."
```

Comment:
I liked the earlier version but there's parts of this that're better. I might split the difference a bit:

```suggestion
Since female octopuses are often larger—and most octopuses are decidedly anti-social—there's always a chance that his lady partner will get bored and/or hungry mid-coitus, give up on the romance, and start snacking away.
```

will get bored and/or hungry, give up on the romance, and instead snack on her lover.

Reply:
Good improvement!